src/lib/bootloader/proposal_client.rb

```diff
@@ -36,7 +36,9 @@ def boot_map
           # TRANSLATORS: kind of boot. It is term for way how x86_64 can boot
           "grub2-efi"    => _("EFI boot"),
           # TRANSLATORS: kind of boot. It is term for way how can boot.
-          "systemd-boot" => _("Systemd boot")
+          "systemd-boot" => _("Systemd boot"),
+          # TRANSLATORS: kind of boot. It is term for way how can boot.
```

Comment:
I would help translators with explanation what is BLS - something like `grub2 adapted to conform Boot Loader Specification (BLS)`.
And that reminds me that I do not like much that BLS boot as Systemd boot is also BLS boot...maybe something like `grub2 with BLS`  can be better?